WordPress/Classes/ViewRelated/Media/CachedAnimatedImageView.swift

```diff
@@ -29,6 +29,17 @@ public class CachedAnimatedImageView: UIImageView, GIFAnimatable {
 
     public var gifPlaybackStrategy: GIFPlaybackStrategy = MediumGIFPlaybackStrategy()
```

Comment:
Since the new `gifStrategy: GIFStrategy` is public, maybe we can make `gifPlaybackStrategy: GIFPlaybackStrategy` private?